com.hack_computer.Clubhouse.json

```
@@ -0,0 +1,276 @@
+{
+    "app-id": "com.hack_computer.Clubhouse",
+    "add-extensions": {
+        "com.hack_computer.Clippy.Extension": {
+            "version": "master",
+            "directory": "clippy",
+            "no-autodownload": false,
+            "autodelete": true
+        }
+    },
+    "branch": "master",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
```

Updade runtime to 3.38
```suggestion
    "runtime-version": "3.38",
```

---

This app doesn't work yet with 3.38.